tests/pools/cluster/dynamic.test.js

```diff
@@ -80,6 +82,8 @@ describe('Dynamic cluster pool test suite ', () => {
     await TestUtils.waitExits(longRunningPool, max - min)
     // Here we expect the workers to be at the max size since that the task is still running
     expect(longRunningPool.workers.length).toBe(min)
+    // we need to clean up the resources after our test
```

Comment:
Nitpick: comment shall start with a capital letter ;)

Reply:
Got it, done